wordpress.tv/public_html/wp-content/themes/wptv2/style.css

```diff
@@ -1577,7 +1577,7 @@ div.sharedaddy.sd-rating-enabled .sd-like .post-likes-widget, div.sharedaddy.sd-
 }
 
 #commentform p {
-	font-size: 10px;
+	font-size: 16px;
```

Comment:
Hi, since this PR hasn’t been merged yet, I wanted to suggest avoiding setting font sizes in pixels, instead opting for `rem` units (i.e., `font-size: 1rem;`).

Reply:
I would usually agree. But since the whole CSS for the theme is always using `px` values for `font-size`, I would not use relative units for this change.

There is an initiative to create a completely new WordPress.tv theme and in this one relative units will be used.